### ISSUE TITLE
Fix sphinx warnings

### ISSUE THREAD

**Issue by DrDaveD** [apptainer/apptainer-userdocs#46]
Doing 'make html' or 'make epub' prints a bunch of warnings that are currently ignored.  They can also be seen in CI runs.  Fix the warnings.

**Comment by pedroalvesbatista** [comment]
I suppose those are contained at `Makefile` and `cli` folder, correct ?

**Comment by DrDaveD** [comment]
No.  Try running 'make html' in your workspace.  It does require having sphinx installed on your machine and a copy of the https://github.com/apptainer/apptainer source in an `apptainer_source` subdirectory.  Or look at the detailed log of a CI run from any pull request in this repository.

**Comment by pedroalvesbatista** [comment]
> No. Try running 'make html' in your workspace. It does require having sphinx installed on your machine and a copy of the https://github.com/apptainer/apptainer source in an `apptainer_source` subdirectory. Or look at the detailed log of a CI run from any pull request in this repository.

I tried that, and there is a need to run `git submodule init` and then `git submodule update` to fetch `apptainer_source`. 

And agree, there are a bunch or warnings that are kind of unnecessary or at least little informative. So, the best way now would be to decide which message we should provide, is that it ?

**Comment by DrDaveD** [comment]
Yes, decide which messages we should fix.

**Comment by symgryph** [comment]
This also fixes 'mconfig' missing. Probably need to update the readme "git submodule init && git submodule update && make epub
